src/components/routeComponents/markAttendance/RoomAttendanceGrid.jsx

```diff
@@ -110,8 +114,16 @@ const RoomAttendanceGrid = ({
 				/>
 				<StyledGridAttendanceBottomSection>
 					<StyledGridAttendanceCount>
-						<div>{`${presentCount} Present | ${absentCount} Absent`}</div>
-						<StyledGridRecordCount>{`Total records: ${totalCount}`}</StyledGridRecordCount>
+						<div>
+							{presentIds.length}
+							{PRESENT}
```

Comment:
Add the pipe separator here
```suggestion
							{PRESENT}  |
```